addons/l10n_vn/data/template/account.tax-vn.csv

```diff
@@ -1,33 +1,49 @@
+"","","","","","","","","base","refund","-Untaxed Purchase of Goods and Services taxed 10%",,,
+"","","","","","","","","tax","refund","-VAT on purchase of goods and services 10%","chart1331",,
+"tax_purchase_vat5","5%","Deductible VAT 5%","Deductible VAT 5%","5.0","percent","purchase","tax_group_5","base","invoice","+Untaxed Purchase of Goods and Services taxed 5%",,"False","Thuế GTGT được khấu trừ 5%"
+"","","","","","","","","tax","invoice","+VAT on purchase of goods and services 5%","chart1331",,
+"","","","","","","","","base","refund","-Untaxed Purchase of Goods and Services taxed 5%",,,
+"","","","","","","","","tax","refund","-VAT on purchase of goods and services 5%","chart1331",,
+"tax_purchase_vat0","0%","Deductible VAT 0%","Deductible VAT 0%","0.0","percent","purchase","tax_group_0","base","invoice","+Untaxed Purchase of Goods and Services taxed 0%",,"False","Thuế GTGT được khấu trừ 0%"
+"","","","","","","","","tax","invoice",,,,
+"","","","","","","","","base","refund","-Untaxed Purchase of Goods and Services taxed 0%",,,
+"","","","","","","","","tax","refund",,,,
+"tax_purchase_vat_exemption","VAT EXEMPTION","VAT Exemption","VAT EXEMPTION","0.0","percent","purchase","tax_group_exemption","base","invoice","+Untaxed Purchase of Goods and Services taxed VAT Exemption",,"False","Không thuộc đối tượng chịu thuế GTGT"
+"","","","","","","","","tax","invoice",,,,
+"","","","","","","","","base","refund","-Untaxed Purchase of Goods and Services taxed VAT Exemption",,,
+"","","","","","","","","tax","refund",,,,
+"tax_purchase_import_10","10% Import","Deductible VAT 10% For Imported Goods","Deductible VAT 10% For Imported Goods","10.0","percent","purchase","tax_group_10","base","invoice","+tax_purchase_import_10_base",,"False",
+"","","","","","","","","tax","invoice","+tax_purchase_import_10","chart33312",,
+"","","","","","","","","base","refund","-tax_purchase_import_10_base",,,
+"","","","","","","","","tax","refund","-tax_purchase_import_10","chart33312",,
+"tax_purchase_import_5","5% Import","Deductible VAT 5% For Imported Goods","Deductible VAT 5% For Imported Goods","5.0","percent","purchase","tax_group_5","base","invoice","+tax_purchase_import_5_base",,"False",
+"","","","","","","","","tax","invoice","+tax_purchase_import_5","chart33312",,
+"","","","","","","","","base","refund","-tax_purchase_import_5_base",,,
+"","","","","","","","","tax","refund","-tax_purchase_import_5","chart33312",,
+"tax_purchase_import_0","0% Import","Deductible VAT 0% For Imported Goods","Deductible VAT 0% For Imported Goods","0.0","percent","purchase","tax_group_0","base","invoice","+tax_purchase_import_0_base",,"False",
+"","","","","","","","","tax","invoice",,,,
+"","","","","","","","","base","refund","-tax_purchase_import_0_base",,,
+"","","","","","","","","tax","refund",,,,
+"tax_import","Import Tax","Import Tax","Import Tax","5.0","percent","purchase","tax_group_import","base","invoice","+tax_import_base",,"True",

```

Comment:
The names are a bit confusing.
There is two 5% import tax, but one is the import tax and the other isn't?